opentelemetry-stdout/Cargo.toml

```diff
@@ -26,8 +26,8 @@ async-trait = { workspace = true, optional = true }
 chrono = { version = "0.4.34", default-features = false, features = ["now"] }
 thiserror = { workspace = true, optional = true }
 futures-util = { workspace = true, optional = true }
-opentelemetry = { version = "0.23", path = "../opentelemetry", default_features = false }
-opentelemetry_sdk = { version = "0.23", path = "../opentelemetry-sdk", default_features = false }
+opentelemetry = { version = "0.23", path = "../opentelemetry", default-features = false }
```

Comment:
given the default is now including all signals, wondering if we can actually remove this `default-features=false` completely?

Reply:
+1

Reply:
Updated.